design-documents/graph-ql/coverage/Wishlist.graphqls

```diff
@@ -17,11 +13,11 @@ type Customer {
 
 type Wishlist {
     id: ID
-    items: [WishlistItem]
+    items: [WishlistItemInterface]
     items_count: Int
     sharing_code: String
     updated_at: String
-    #name: String should be added in Commerce edition
+    name: String @doc(description: "Avaialble in Commerce edition only")
```

Comment:
Can we add this field thought the dedicated module?

Reply:
This is just a design, we don't have modules here. What do you mean?

In general, yes, fields can be added from separate modules.